lib/rubocop/target_finder.rb

```diff
@@ -5,6 +5,8 @@ module RuboCop
   # and picking ruby files.
   # @api private
   class TargetFinder
+    HIDDEN_FILE_PATTERN = "#{File::SEPARATOR}."
```

Comment:
I'm not sure that naming this "pattern" is a good idea, as it makes me think of regular expressions. :-)

Reply:
updated to `HIDDEN_PATH_SUBSTRING`